client/app/bundles/course/assessment/submission/submissions.js

```diff
@@ -1,5 +1,15 @@
+/* eslint-disable global-require */
 import initializeDownloadLink from 'lib/helpers/initializeDownloadLink';
 
-require('./submission');
+function initialize() {
+  // Initializers
+  require('./submission');
+  initializeDownloadLink('.btn.download');
+}
 
-initializeDownloadLink('.btn.download');
+Promise.all([
```

Comment:
Possible to only load these libraries when the scribing answers is rendered ?  Because most of the submissions would not have scrbing answers, we should not delay the page load to downloading the libraries.

Reply:
Ok